spec/services/stripe_model_callbacks/invoice/updated_service_spec.rb

```diff
@@ -0,0 +1,65 @@
+require "rails_helper"
+
+describe StripeModelCallbacks::Invoice::UpdatedService do
+  subject(:updated_service) { described_class.execute!(event: stripe_event) }
+
+  let(:stripe_event) { Stripe::Event.construct_from(event_values) }
+  let(:event_payload) { File.read("spec/fixtures/stripe/invoice/created.json") }
+  let(:event_values) do
+    {
+      id: "evt_1GiGbaGJAmuZ9ZZZbKpF4hGj",
+      type: event_type,
+      data: JSON.parse(event_payload, symbolize_names: true)
+    }
+  end
+  let(:stripe_id) { "in_1GxxxxxxZ9ZZZxxXLFQx" }
+
+  shared_examples "do not create new invoice" do
```

Comment:
This makes the code very abstract. Now the reader has to jump up and down to grasp whatever the test actually does :-)

https://www.youtube.com/watch?v=oO-FMAdjY68

Reply:
Fixed :)